synapse/tests/test_cortex.py

```diff
@@ -2623,6 +2623,9 @@ class Bawk:
             await _test('$(2 != 1)', 1)
             await _test('$(2 = 1)', 0)
             await _test('$(2 = 2)', 1)
+            await _test('$(02 = 2)', 1)
+            await _test('$(2 = 02)', 1)
+            await _test('$(2 = 2.0)', 1)
```

Comment:
need an 0x2000 case and an octal case that matters :)

Reply:
I'm gonna defer that, since we don't currently support parsing bare hex or octal constants at all.